rust/libnewsboat/src/utils.rs

```diff
@@ -1037,6 +1037,33 @@ pub fn locale_to_utf8(text: &[u8]) -> String {
     String::from_utf8(converted).expect("convert_text() returned a non-UTF-8 string")
 }
 
+/// Parses a string in the form `author@example.com (Example Author)`
+/// into a pair of (author, email).
+/// If the pattern cannot be matched or the name is empty, the input
+/// is returned completely as author with empty email.
+///
+/// Invalid data (outside of utf-8) is dropped
```

Comment:
This is a bit pedantic of me, but I think it's important that the data is replaced, not dropped; there's a difference between getting an empty string and getting something like "������":

```suggestion
/// Invalid data (outside of utf-8) is replaced with Unicode replacement character.
```

Reply:
Good point, done